test/stream/api/sink/statistics-total-001-ok/001-statistics-total/expected-http-code

```diff
@@ -0,0 +1 @@
+200
```

Comment:
We should check that `source` and `sink` exists. 

I wouldn't be happy if I made a typo in the URL, and instead of a 404 it returns empty statistics.

Reply:
There are methods, in the definition repository `<repo>.ExistsOrErr(...)`.

The error can be directly returned from the API service, it has correct status code.